packages/shared_preferences/shared_preferences_windows/example/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
We should either have a license, or remove the file; it looks like both have been done. I'm not sure if one or the other is the preferred practice going forward so someone who knows should weigh in.

Reply:
I just copied the license from the front-end package